react-backend/shared/middleware/error.js

```diff
@@ -12,7 +14,13 @@ exports.notFound = (req, res, next) => {
 };
 
 // error handler
-exports.errorHandler = (err, req, res) => {
+exports.errorHandler = (err, req, res, next) => { // eslint-disable-line
+  if (Boom.isBoom(err)) {
+    const { statusCode, message } = err.output.payload;
+
+    return res.status(statusCode).send({ result: null, error: message });
+  }
+
   if (err === 'notauthorized') {
```

Comment:
Is this still needed if we're using `Boom`?

Reply:
I didn't look through all the other routes, there may still be something that relies on this. I'll only remove it once all the other routes have been brought into line.